bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -65,6 +65,20 @@ GradleBuilder.prototype.getArgs = function(cmd, opts) {
     return args;
 };
 
+/*
+ * This returns a promise
+ */
+
+GradleBuilder.prototype.runGradleWrapper = function(gradle_cmd) {
+	var gradlePath = path.join(this.root, 'gradlew');
+	if(fs.existsSync(gradlePath)) {
+		;
```

Comment:
I spent an hour trying to get this stupid code to work in Windows.  This is so we don't keep running the wrapper task every time we build, clean or whatever else.  It looks weird because fs.existsSync kept forever returning true.